oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/endpoint/OAuth2AuthorizationResponse.java

```diff
@@ -63,6 +65,15 @@ public String getCode() {
 		return this.code;
 	}
 
+	/**
+	 * Returns the nonce.
+	 *
+	 * @return the nonce
+	 */
+	public String getNonce() {
```

Comment:
The `nonce` does not come back in the Authorization Response. It comes back in the ID Token. Please remove.